joblib/test/test_pool.py

```diff
@@ -18,7 +16,9 @@
     from joblib.pool import reduce_memmap
 
 
-TEMP_FOLDER = None
+@fixture(scope='function')
+def tmpdir_path(tmpdir):
+    return tmpdir.strpath
```

Comment:
`tmpdir` is a fixture provided by pytest - it is a `LocalPath` object which gives a unique path for each test. I found it beeter to deal with a string to I wrapped it in another fixture ! Now we don't need to care about setup and teardown anymore.